README.md

```diff
@@ -92,6 +92,52 @@ gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys <last eight symbols of
 gpg --keyserver hkp://keyserver.ubuntu.com --send-keys <last eight symbols of gnupg keyId>
 ```
 
+### Stencil Server API
```

Comment:
The heading could be `Hosting descriptors behind an HTTP Server`

Reply:
These APIs are not just for hosting the descriptor sets, but also for storing some related metadata like version.

Reply:
`Managing descriptors behind an HTTP Server` would this suit?